src/stable/test/jest_link.ts

```diff
@@ -1,16 +1,26 @@
 import { execSync } from 'child_process';
-import { join, resolve } from 'path';
+import { join } from 'path';
+
```

Comment:
Let's get rid of all of this machinery across the entire PR, because the `npm` tests during the release process should do essentially the same thing.

Reply:
I'm going to do this last of all because I need it to test everything about this pr

Reply:
I also added some pack hack to assertAzleExperimentalDeps that will need to be cleaned up

Reply:
I am wondering if it would be wise to at least leave the plumbing around in the workflow. At least while I am testing this it is very handy to be able to turn it on when I need.

Reply:
Though by that same argument maybe we should leave it here too? This isn't exactly production code that is going in people's canisters. We don't have to run the pack mode regularly, but maybe it would be nice to have it around just in case?

Reply:
Please remove once these removal comments are the last things